plugin/vim-erlang-tags.vim

```diff
@@ -53,9 +53,13 @@ function! VimErlangTags()
     endif
 endfunction
 
-function! AsyncVimErlangTags()
+function! AsyncVimErlangTags(...)
+    let param = ""
```

Comment:
```suggestion
    let param = join(a:000, " ")
```

Reply:
Github has this magic new "suggestion" thing but it doesn't appear to work across multiple lines... This was meant to replace the for loop.

Reply:
Ah, cool one, will do. Sorry, I'm just learning vimscript now, didn't know that function 😄

Reply:
Hah and now that I've complained about it, GH just showed me how to apply the suggestion thing to multiple lines. Neato.